tests/tasks/Cloudwatch/cw-metric.yaml

```diff
@@ -0,0 +1,21 @@
+apiVersion: tekton.dev/v1beta1
+kind: Task
+metadata:
+  name:  cloudwatch
+  namespace: tekton-pipelines
+spec:
+  description: "cw task to push metrics relying on generate task."
+  params:
+    - name: region
+      default: "us-west-2"
+    - name: desired-nodes
+    - name: load-result
+  steps:
+  - name: cw-emit
+    image: alpine/k8s:1.22.6
+    script: |
+      if [[ "$(params.load-result)" -eq 0 ]]; then
+      aws cloudwatch --region $(params.region) put-metric-data --metric-name $(params.desired-nodes)_nodes --namespace Cl2Load-Tests --unit Count --value 1 --dimensions TypeOfTest=Cl2_Test
+      else
+      aws cloudwatch --region $(params.region) put-metric-data --metric-name $(params.desired-nodes)_nodes --namespace Cl2Load-Tests --unit Count --value 0 --dimensions TypeOfTest=Cl2_Test 
```

Comment:
`$(params.desired-nodes)_nodes` should not be the`metric-name`. Number of nodes is a dimension. The `metric-name` can be something like `success`.
Also why do we want dimension of `TypeOfTest=Cl2_Test`, do we expect other type of tests?
@hakuna-matatah WDYT? We should get this right. It'd be harder to change it later.

Reply:
After performing some tests ,i come up with that name ,Will be changing accordingly as per your suggestions.

Reply:
We can have metric name as `cl2-loadtest` and dimension as nodes and namespace would be `k8s version`
In future we would be running different types of tests like density etc and we need to differentiate these types of test with `loadtests` we are running today.